docs/sftt/index.md

```diff
@@ -6,6 +6,6 @@ In this section you will find all the developer documentation related to our Spe
 
 The full API specification for our backend. This page lists every backend route and describes what it does, the request body and the possible responses. 
 
-## [ArcGIS Map Documentation](./arcgis-architecture.md)
+## [(DEPRICATED) ArcGIS Map Documentation](./arcgis-architecture.md)
```

Comment:
Do you think we need docs for google maps?

Reply:
I think it would be good to have some. The biggest reason we had ArcGIS docs was because we were storing data in one of their databases instead of on our own backend. 

idk enough about the Google maps api to write documentation about it.